README.md

```diff
@@ -69,15 +68,15 @@ Follow the below steps to deploy the following into a clean account. This will b
 * Sonar instance
 
 1. Generate and configure repos (default includes front-end, API, IaC for Jenkins and ECS. If you're feeling bold you can try the *experimental* approach below this.)
-    1. Fork, clone and cd into https://github.com/excellaco/tcp-glue
+    1. Fork, clone and cd into https://github.com/excellaco/tcp-glue [Why do they need to fork it? They won't be changing tcp-glue.]]
```

Comment:
Correct, they don't need to fork. If using xg, it creates the repos with new names, which is what we want (whether using xg or not). So if not using xg, after cloning they are going to have to rename and point them to new githubs. What do you think about going full on xg?

Reply:
OK; I'll remove the comment. I think xg-only at some point might be a worthwhile goal.